srcpkgs/capnproto/template

```diff
@@ -10,7 +10,7 @@ maintainer="eater <hello@eaterofco.de>"
 license="MIT"
 homepage="https://capnproto.org"
 distfiles="${homepage}/capnproto-c++-${version}.tar.gz"
-checksum=d1f40e47574c65700f0ec98bf66729378efabe3c72bc0cda795037498541c10d
+checksum=83680aaef8c192b884e38eab418b8482d321af6ae7ab7befa3a9370b8e716aad
 
 CXXFLAGS="-std=c++14"
```

Comment:
Can you check if it builds after removing this flag?

Reply:
It works locally at least.